### PR TITLE
HEAT-236-fix: Updated the Expression

### PR DESCRIPTION
**Pull request by KalyanaVadlamani**

### CODE REVIEW

File: charts/generic-prometheus-alerts/templates/ingress-alerts.yaml [hunk 1]
```diff
@@ -28,18 +28,54 @@ spec:
...
+        #     dashboard_url: {{ $.Values.grafanaUrl }}/d/{{ $.Values.dashboardId }}/kubernetes-ingress-traffic?{{ $.Values.orgId }}&var-namespace={{ $targetNamespace }}&var-ingress={{ $targetIngress }}&from=now-5m&to=now
+        #   expr: |-
+        #     avg(rate(nginx_ingress_controller_requests{exported_namespace=~"{{ $targetNamespace }}", ingress=~"{{ $targetIngress }}", path="/health", status=~"5.*"}[{{ .Values.ingress5xxErrorWindowMinutesHealthEndpoint }}m]) > 0.01) by (ingress, exported_namespace)
+        #     {{ $businessOrAllHoursExpression }}
+        #   for: 5m
+        #   labels:
+        #     severity: {{ .Values.alertSeverity }} 
         - alert: 5xxErrorResponsesOnHealthEndpoint
           annotations:
             message: High rate of 5xx errors detected on /health path in Ingress {{`{{`}} $labels.exported_namespace {{`}}`}}/{{`{{`}} $labels.ingress {{`}}`}}.
```
Comment (mattops) [inline charts/generic-prometheus-alerts/templates/ingress-alerts.yaml:69]:
```suggestion
            message:  5xx errors for /health endpoint.  Counted {{ $value }} within {{ .Values.ingress5xxErrorWindowMinutesHealthEndpoint }} min time frame.  Ingress resource {{`{{`}} $labels.exported_namespace {{`}}`}}/{{`{{`}} $labels.ingress {{`}}`}}.
```

Comment (KalyanaVadlamani) [reply charts/generic-prometheus-alerts/templates/ingress-alerts.yaml]: Updated & resolved

File: charts/generic-prometheus-alerts/values.yaml [hunk 2]
```diff
@@ -128,5 +128,8 @@ ingress5xxErrorWindowMinutes: 1
 # Set the error window minutes for 5xx ingress High5xxErrorRateOnHealthEndpoint
 ingress5xxErrorWindowMinutesHealthEndpoint: 5
 
+# Threshold for number of instances of 5xx ingress High5xxErrorRateOnHealthEndpoint
+ingress5xxErrorCountThreshold: 10
```
Comment (mattops) [inline charts/generic-prometheus-alerts/values.yaml:132]:
```suggestion
ingress5xxErrorCountThresholdHealthEndpoint: 10
```

File: charts/generic-prometheus-alerts/values.yaml [hunk 3]
```diff
@@ -128,5 +128,8 @@ ingress5xxErrorWindowMinutes: 1
 # Set the error window minutes for 5xx ingress High5xxErrorRateOnHealthEndpoint
 ingress5xxErrorWindowMinutesHealthEndpoint: 5
 
+# Threshold for number of instances of 5xx ingress High5xxErrorRateOnHealthEndpoint
```
Comment (petergphillips) [inline charts/generic-prometheus-alerts/values.yaml:131]:
```suggestion
# Threshold for number of instances of 5xx ingress 5xxErrorResponsesOnHealthEndpoint
```

**Pull request closed without merging**